webrender/src/platform/windows/font.rs

```diff
@@ -278,22 +278,24 @@ impl FontContext {
 
         let mut pixels = analysis.create_alpha_texture(texture_type, bounds);
 
-        let lut_correction = match glyph_options {
-            Some(option) => {
-                if option.force_gdi_rendering {
-                    &self.gdi_gamma_lut
-                } else {
-                    &self.gamma_lut
-                }
-            },
-            None => &self.gamma_lut
-        };
-
-        lut_correction.preblend_rgb(&mut pixels, width, height,
-                                    ColorLut::new(key.color.r,
-                                                  key.color.g,
-                                                  key.color.b,
-                                                  key.color.a));
+        if render_mode != FontRenderMode::Mono {
```

Comment:
This will run the gamma correct on alpha and subpixel modes (just not AA disabled). Is that intentional or is it meant to only be on subpixel mode?